services/blockchain-app-registry/tests/functional/mysqlKVStore.test.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const keyValueTable = require('../../shared/database/mysqlKVStore');
+
+const KEY_1 = 'KEY_1';
+const KEY_2 = 'KEY_2';
+const VALUE_1 = 'VALUE_1';
+const VALUE_2 = 'VALUE_2';
+
+afterEach(async () => {
+	await keyValueTable.delete(KEY_1);
+	await keyValueTable.delete(KEY_2);
+});
+
+describe('Test set and get methods', () => {
+	it('Sets and updates value correctly', async () => {
```

Comment:
As a general practice, all the testcase descriptions must be declarative and begin with `it('should .....', ...)` (in small case).
Let us please ensure that we stick to this standard henceforth.

Reply:
we did not use to follow the `should..` format before. refactored the tests and will follow.

Reply:
Yes, I noticed it pretty late. So, made a comment to ensure that we stick to it in the future starting now 🙂
Small changes!!